presto-main/src/main/java/io/prestosql/server/PluginClassLoader.java

```diff
@@ -16,8 +16,6 @@
 import com.google.common.collect.ImmutableList;
 
```

Comment:
Commit message is invalid:

You says that: `Use platform classloader in Java 8 and 11 for plugins`, while we were already using platform classloader for java 9 and so I guess it was working as well for java 11.